specs/phase0/p2p-interface.md

```diff
@@ -601,6 +620,60 @@ Clients MUST support requesting blocks since the latest finalized epoch.
 
 Clients MUST respond with at least one block, if they have it. Clients MAY limit the number of blocks in the response.
 
+#### Ping
+
+**Protocol ID:** `/eth2/beacon_chain/req/ping/1/`
+
+Request Content:
+
+```
+(
+  uint64
+)
+```
+
+Response Content:
+
+```
+(
+  uint64
+)
+```
+
+Sent intermittently, the `Ping` protocol checks liveness of connected peers.
+Peers send and respond with their local metadata sequence number (`MetaData.seq_number`).
+
+If the peer does not respond to the `Ping` request, the client MAY disconnect from the peer.
+
+A client can then determine if their local record of a peer's MetaData is up to date
+and MAY request an updated version via the `MetaData` RPC method if not.
+
+The request MUST be encoded as an SSZ-field.
+
+The response MUST consist of a single `response_chunk`.
+
+#### GetMetaData
+
+**Protocol ID:** `/eth2/beacon_chain/req/metadata/1/`
+
+No Request Content.
```

Comment:
Interesting, the first RPC call without a request content. Does that mean an empty payload, but still a request header with a 0-length in it? Or no bytes at all?

Reply:
This came from @AgeManning in his previous PR. 
I interpreted it as no bytes at all

Reply:
Think we can spec it both ways, let's ask on the call

Reply:
Talked to age, was the intention and works on libp2p...

Will change it later if Raul chimes in otherwise